feature_extraction.py

```diff
@@ -140,3 +140,215 @@ def transform(self, X):
                         how='left')
 
         return {'numerical_features': X[self.groupby_aggregations_names].astype(np.float32)}
+
+
+class Application(BaseTransformer):
+    def __init__(self):
+        super().__init__()
+
+    @property
+    def application_names(self):
+        return ['ANNUITY_INCOME_PERCENTAGE',
+                'CREDIT_TO_GOODS_RATIO',
+                'DAYS_EMPLOYED_PERCENTAGE',
+                'EXT_SOURCES_MEAN',
+                'INCOME_CREDIT_PERCENTAGE',
+                'INCOME_PER_PERSON',
+                'PAYMENT_RATE']
+
+    def transform(self, X, y=None):
+        X['DAYS_EMPLOYED'].replace(365243, np.nan, inplace=True)
```

Comment:
@pknut I think the cleane-up where we deal with missing values outliers etc should be done in a seperate step (or steps)